### PR TITLE
Deployment restructure

### PR DESCRIPTION
**Pull request by thehonker**
This PR:

- Creates Dockerfiles for nginx and php-fpm
- Creates docker-compose stacks for prod and dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+
+See [https://github.com/wetfish/production-manifests](https://github.com/wetfish/production-manifests)
+for production deployment and full stack dev env info.
+
+For development, to run just this stack, do 
+```bash
+cp mariadb.env.example mariadb.env
+# -> edit, change passwords and other info as needed
+cp php.env.example php.env
+# -> edit, change passwords to match
```
Comment (itsrachelfish) [inline README.md]: This project doesn't use mariadb. I'm assuming this was copied from other projects?

**Review by itsrachelfish — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (thehonker) [reply README.md]: Yep, and it looks like we don't need that line about php.env either?

Comment (itsrachelfish) [reply README.md]: Yep, there is no local config for this project, it's nearly all client side JS with a bit of PHP to glitch things up

Comment (thehonker) [reply README.md]: Should be resolved by [https://github.com/wetfish/glitch/pull/2/commits/c485280cc32804e73a784ef5aa054a212155abba](https://github.com/wetfish/glitch/pull/2/commits/c485280cc32804e73a784ef5aa054a212155abba)

**Review by itsrachelfish — APPROVED** [review]
Tested locally, working as expected :+1: 

image

**Pull request merged**